game/scripts/npc/items/item_dagon.txt

```diff
@@ -68,12 +68,12 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_intellect"                                 "15 18 21 24 27 35 50 70 90"
+        "bonus_intellect"                                 "15 18 21 24 27 40 60 90 130"
       }
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_all_stats"                                 "5"
+        "bonus_all_stats"                                 "5 5 5 5 5 10 15 25 30"
```

Comment:
Isn't this 5 at all levels on purpose? I don't like this at all.

Reply:
I dont see why this items stats shouldn't scale. Every other item in the game has scaling stats.